cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1180,7 +1185,7 @@ private KafkaRebalanceAnnotation rebalanceAnnotation(Reconciliation reconciliati
                     KafkaRebalanceAnnotation.none : KafkaRebalanceAnnotation.valueOf(rebalanceAnnotationValue);
         } catch (IllegalArgumentException e) {
             rebalanceAnnotation = KafkaRebalanceAnnotation.unknown;
-            LOGGER.warnCr(reconciliation, "Wrong annotation value {}={} on {}/{}",
+            LOGGER.warnCr(reconciliation, "Wrong annotation value {}={} on {}/{}. The rebalance resource supports the following annotations : approve, refresh, stop.",
```

Comment:
Don't use space before `:`:

```suggestion
            LOGGER.warnCr(reconciliation, "Wrong annotation value {}={} on {}/{}. The rebalance resource supports the following annotations: approve, refresh, stop.",
```